app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsImageCard.test.tsx.snap

```diff
@@ -18,10 +18,14 @@ exports[`ListingDetailsImageCard displays image card when no tag 1`] = `
         className="image-card__inner"
       >
         <img
-          alt="A picture of the building"
```

Comment:
would rather keep the old `alt` values. a user with a screen reader will already know the title of the listing. ideally we would have real descriptions for the listing images e.g. 'a three story building with mid century modern architecture and a curved walkway surrounded with yucca plants and shrubs'. however, we're not going to get that any time soon. another option is to leave off the alt rather than include redundant info which will be noisy for a11y users